packages/client/src/pages/settings/profile.vue

```diff
@@ -122,68 +115,32 @@ function changeBanner(ev) {
 }
 
 async function editMetadata() {
-	const { canceled, result } = await os.form(i18n.ts._profile.metadata, {
-		fieldName0: {
+	const meta = await os.api('meta');
```

Comment:
I think this can be replaced with `instance` from `import { instance } from '@/instance';`